api/src/pcapi/core/offerers/synchronize_venues_banners_with_google_places.py

```diff
@@ -145,33 +145,34 @@ def synchronize_venues_banners_with_google_places(
     images_ignored_due_to_ratio = []
     banner_synchronized_venue_ids = set()
     for venue in venues_without_photos:
-        if venue.googlePlacesInfo:
-            if (datetime.datetime.utcnow() - venue.googlePlacesInfo.updateDate).days > 62:
-                continue
-        else:
-            place_id = get_place_id(venue.name, venue.address, venue.city, venue.postalCode)
-            if not place_id:
-                continue
-            venue.googlePlacesInfo = offerers_models.GooglePlacesInfo(placeId=place_id)
-
-        nb_places_found += 1
-        place_details = get_place_photos_and_owner(venue.googlePlacesInfo.placeId)
-        if not (place_details and place_details.photos):
-            continue
-        nb_places_with_photo += 1
-
-        owner_photo = get_owner_photo(place_details.photos, place_details.name)
-        if owner_photo:
-            photo = owner_photo
-            photo_prefix = "owner"
-            nb_places_with_owner_photo += 1
-        else:
-            photo = place_details.photos[0]
-            photo_prefix = ""
         try:
+            if venue.googlePlacesInfo:
+                if (datetime.datetime.utcnow() - venue.googlePlacesInfo.updateDate).days > 62:
+                    continue
+            else:
+                place_id = get_place_id(venue.name, venue.address, venue.city, venue.postalCode)
+                if not place_id:
+                    continue
+                venue.googlePlacesInfo = offerers_models.GooglePlacesInfo(placeId=place_id)
+
+            nb_places_found += 1
+            place_details = get_place_photos_and_owner(venue.googlePlacesInfo.placeId)
+            if not (place_details and place_details.photos):
+                continue
+            nb_places_with_photo += 1
+
+            owner_photo = get_owner_photo(place_details.photos, place_details.name)
+            if owner_photo:
+                photo = owner_photo
+                photo_prefix = "owner"
+                nb_places_with_owner_photo += 1
+            else:
+                photo = place_details.photos[0]
+                photo_prefix = ""
             venue.googlePlacesInfo.bannerUrl = save_photo_to_gcp(venue.id, photo, photo_prefix)
             venue.googlePlacesInfo.bannerMeta = photo.model_dump()
             banner_synchronized_venue_ids.add(venue.id)
+            db.session.commit()
```

Comment:
Si tu commites ici, il faut aussi appeler `db.session.rollback()` dans les 2 branches `except`. Sinon, les modifications qui auront été faites (par exemple `venue.googlePlacesInfo = offerers_models.GooglePlacesInfo(placeId=place_id)`) pour ce lieu resteront dans la session et seront commitées lors de l'appel à `commit()` réalisé pour le prochain lieu de la bouche `for`.